.github/workflows/build-browser.yml

```
@@ -114,8 +114,8 @@ jobs:
          fi
 
 
-  build:
-    name: Build
+  build-source:
+    name: Build Browser Source
```

```suggestion
    name: Build browser source
```

⛏️ I treat these like statements vs. titles. Applies to all your changes in this regard.